RunningOrder/InlineEditableList.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  InlineTexField.swift
+//  RunningOrder
+//
+//  Created by Lucas Barbero on 20/07/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+import AppKit
+
+/// Display an inline editable list of string element
+struct InlineEditableList: View { // Change name ???
+
+    let title: String
+
+    let placeholder: String
+
+    @Binding var values: [String]
+    @State private var hovered = false
+
+    var body: some View {
+        VStack {
+
+            HStack {
+                Text(title).foregroundColor(.gray).padding(.all, 6)
+                Spacer()
+                RoundButton(imageName: NSImage.addTemplateName, color: Color.accentColor, action: addTextfieldValue)
+                    .frame(width: 20, height: 20)
+                    .padding(.horizontal, 10)
+            }
+
+            ForEach(values.indices, id: \.self) { index in
+                ZStack(alignment: .trailing) {
```

Comment:
why trailing ?

Reply:
In order to stuck the button to the right of the ZStack, the textfield will take all the space it can, so it doesn't change anything for it

Reply:
ok